packages/patternfly-4/react-table/src/components/Table/utils/decorators/headerCol.tsx

```diff
@@ -1,8 +1,8 @@
 import * as React from 'react';
 import { IExtra, IFormatterValueType, ITransform } from '../../Table';
 
-export const headerCol = (id: string = 'simple-node') => {
-  // tslint:disable-next-line:no-shadowed-variable
+export const headerCol = (id = 'simple-node') => {
+  // eslint-disable-next-line no-shadow
   const headerCol: ITransform = (value: IFormatterValueType, { rowIndex }: IExtra = {}) => {
```

Comment:
Renaming this variable is probably wise.

Reply:
Done